solana/README.md

```diff
@@ -0,0 +1,93 @@
+# Token Bridge Relayer Solana
+
+## Dependencies
+
+> **Warning**
+> Known to cause problems with Solana version 1.15 - downgrade to Solana 1.14.14
```

Comment:
what are the problems? do they manifest as compile time errors? in any case, it would be great if `make` checked the version of `solana` and errored out early if it’s not 1.14.14